src/data/zmod/basic.lean

```diff
@@ -424,6 +424,13 @@ begin
   rw [zmod.int_coe_eq_int_coe_iff, int.modeq_zero_iff_dvd],
 end
 
+lemma int_coe_eq_int_coe_iff_dvd_sub (a b : ℤ) (c : ℕ) : (a : zmod c) = ↑b ↔ ↑c ∣ a-b :=
+begin
```

Comment:
Starting with `rw int_coe_eq_int_coe_iff` gives an even shorter proof

Reply:
Not sure. The relevant lemma `int.modeq_iff_dvd` swaps a and b, and `int.modeq.symm` is not an iff statement, so I cannot use it to rewrite... Also, `c ∣ a-b ↔ c ∣ b-a` seems to be missing from the library.

Of course, one could change the statement by swapping a and b on the right hand side; then it would just be two rewrites. (And add the statement `c ∣ a-b ↔ c ∣ b-a` to algebra.ring.basic.)

Reply:
I think the fact that `int.modeq_iff_dvd` has `a` and `b` swapped is a good argument to swap the arguments in this lemma too. If we're going to copy the `int.modeq` API across to zmod, we should try to make it match.

> and int.modeq.symm is not an iff statement

Do we have `int.modeq_comm`?

Reply:
OK, I'll swap `a` and `b` in the rhs and replace the proof.
>Do we have `int.modeq_comm`?
`suggest` comes up with `exact eq_comm` for that, but this does not work with `rw`.